RHDEVDOCS-3747 - Added 1.4.2 release notes (#41758)
openshift/openshift-docs: modules/gitops-release-notes-1-4-1.adoc

// Module included in the following assembly:
//
// * gitops/gitops-release-notes.adoc

[id="gitops-release-notes-1-4-1_{context}"]
= Release notes for {gitops-title} 1.4.1

> {gitops-title} 1.4.1 is now available on {product-title} 4.7, 4.8, and 4.9.

Reviewer: it should be 1.4.2

// Module included in the following assembly:
//
// * gitops/gitops-release-notes.adoc

[id="gitops-release-notes-1-4-1_{context}"]
= Release notes for {gitops-title} 1.4.1

{gitops-title} 1.4.1 is now available on {product-title} 4.7, 4.8, and 4.9.

[id="fixed-issues-1-4-1_{context}"]
== Fixed issues

The following issue has been resolved in the current release:

* {gitops-title} Operator v1.4.0 introduced a regression which removes the description fields from `spec` for the following CRDs:

** `argoproj.io_applications.yaml`
** `argoproj.io_appprojects.yaml`
** `argoproj.io_argocds.yaml`

> Before this update, if you try to create an `AppProject` resource using the `"kubectl create"` it failed to synchronize. This update restores the missing description fields in `CustomResourceDefinitions: ArgoCD, AppProject, Application` which can cause `"kubectl create"` to fail. link:https://issues.redhat.com/browse/GITOPS-1721[GITOPS-1721]

Reviewer 2: ```suggestion
Before this update, if you created an `AppProject` resource using the `kubectl create`, it failed to synchronize. This update restores the missing description fields in `CustomResourceDefinitions: ArgoCD, AppProject, Application` that caused `kubectl create` to fail. link:https://issues.redhat.com/browse/GITOPS-1721[GITOPS-1721]
```
